package.json

```diff
@@ -1060,7 +1060,7 @@
                 },
                 {
                     "command": "aws.dev.openMenu",
-                    "when": "aws.isDevMode"
+                    "when": "aws.isDevMode || isCloud9"
```

Comment:
Is this just to make customers lives easier?

Reply:
yes